src/app/ApplicationTemplate.Shared/Business/Impl/AuthenticationService.cs

```diff
@@ -7,10 +7,12 @@
 using System.Threading.Tasks;
 using ApplicationTemplate.Client;
 using MallardMessageHandlers;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 
 namespace ApplicationTemplate.Business
 {
+	[RegisterService(ServiceLifetime.Singleton)]
```

Comment:
It's a cool change overall. Please update the doc at https://github.com/nventive/UnoApplicationTemplate/blob/master/doc/DependencyInjection.md and https://github.com/nventive/UnoApplicationTemplate/blob/master/doc/Configuration.md.

Reply:
Will do